Validation/FlightValidator.cs

```diff
@@ -0,0 +1,118 @@
+﻿using Airport_Ticket_Booking.Domain.FlightManagement;
+using Airport_Ticket_Booking.Domain.General;
+using System.ComponentModel.DataAnnotations;
+using System.Globalization;
+
+
+namespace Airport_Ticket_Booking.Validation
+{
+  
+    public class FlightValidator
+    {
+        public Dictionary<string, string> GetValidationConstraints()
+        {
+            var constraints = new Dictionary<string, string>();
+
+            var properties = typeof(Flight).GetProperties();
+
+            foreach (var property in properties)
+            {
+                var attributes = property.GetCustomAttributes(true);
+                var constraintDescription = new List<string>();
+
+                foreach (var attribute in attributes)
+                {
+                    if (attribute is RequiredAttribute)
+                    {
+                        constraintDescription.Add("Required");
+                    }
+                    else if (attribute is RangeAttribute range)
+                    {
+                        constraintDescription.Add($"Allowed Range: {range.Minimum} → {range.Maximum}");
+                    }
+                    else if (attribute is FutureDateAttribute)
+                    {
+                        constraintDescription.Add("Must be a future date");
+                    }
+                }
+
+                if (constraintDescription.Any())
+                {
+                    constraints[property.Name] = string.Join(", ", constraintDescription);
+                }
+            }
+
+            return constraints;
+        }
+        public List<ValidationResult> ValidateFlight(Flight flight)
```

Comment:
let's have one line space between the different part of the code to have a clean and a more dreadable code. Please apply this to all places in your code.
```suggestion
        }
        
        public List<ValidationResult> ValidateFlight(Flight flight)
```